packages/contentstack/package.json

```diff
@@ -41,10 +41,10 @@
     "@oclif/errors": "^1.3.3",
     "@oclif/plugin-not-found": "^2.3.1",
     "@oclif/plugin-plugins": "^2.1.0",
-    "chalk": "^4.0.0",
+    "chalk": "^5.0.1",
     "debug": "^4.1.1",
     "figlet": "^1.5.2",
-    "inquirer": "^7.1.0",
+    "inquirer": "^8.2.4",
```

Comment:
@netrajpatel We need to test this one. I tested my end prompt working fine without any issue. Release not says it has some major changed in function param structure

V7 => https://github.com/SBoudrias/Inquirer.js/releases/tag/inquirer%407.0.0
V8 => https://github.com/SBoudrias/Inquirer.js/releases/tag/inquirer%408.1.0

Reply:
I've tested with some modules like export-to-csv, clone and it is working fine.
Looks like no breaking changes.